entities/four_year_plan.ts

```diff
@@ -100,6 +100,8 @@ export interface SemesterProps {
   warningPrerequisiteCourses: CourseType[]; // list of courses in with prereq issues
   warningFallvsSpringCourses: CourseType[]; // list of courses with semester issues
   warningDuplicateCourses: CourseType[]; // list of courses in more than one semester
+  year: number; // year number 1,2,3,4,etc.
+  season: string; // Season the semester is (Fall, Winter, Spring, Summer)
```

Comment:
This is little bigger, but season should be an enum.

```ts
enum Season {
  Fall = "Fall",
  Winter = "Winter",
  Spring = "Spring",
  Summer = "Summer",
}
```
This will ensure that the Season value can only be one of the valid values.

Conveniently, the enum values are still strings, so they will format correctly.